docs/UserGuide.md

```diff
@@ -266,44 +309,47 @@ Examples:
 * `tag t/activity i/A420 ta/important ta/NUS ta/schoolwork`
 * `tag t/activity i/A421 ta/important`
 
-### Find results that contain keyword: `find [OPTIONAL_FLAG/]KEYWORD...`
+### Find results that contain keyword: `find [OPTIONAL_PARAMETERS]...`
 
 Searches for doctors, patients and activities that contain the specified keyword as a substring in any of their details.
 If more than 1 keyword is specified, results that contain at least 1 of the keywords will be returned (i.e. `OR` search)
 E.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 
-Users can specify additional optional flags to limit the keyword matching to the fields that is associated with
-each flag. Flags will only match results that contain the associated field (E.g. n/Hans will not return any `activities` 
+Users can specify additional optional parameters to limit the keyword matching to the fields that is associated with each
+flag. Parameters will only match results that contain the associated field (E.g. n/Hans will not return any `activities`
 since `activities` do not have a `name` field.)
 
-The flags are:
+Format: `find [OPTIONAL_PARAMETERS]...`
+
+The parameters are:
+
 * `n/NAME`: Matches the name field (Valid for: `Patients`, `Doctors`)
 * `c/CONTACT_NUMBER`: Matches the contact number field (Valid for: `Patients`, `Doctors`)
 * `a/AGE`: Matches the age field (Valid for: `Patients`)
 * `g/GENDER`: Matches the gender field; The only valid keywords for this field are `M/F/O` (Valid for: `Patients`)
 * `h/HEIGHT`: Matches the height field (Valid for: `Patients`)
 * `w/WEIGHT`: Matches the weight field (Valid for: `Patients`)
-* `b/BLOOD_TYPE`: Matches the blood type field; The only valid keywords for this field are `A/B/AB/O` 
-(Valid for: `Patients`)
-* `m/MEDICAL_CONDITION`: Limits the keyword search to the list of medical conditions of a patient (Valid for: `Patients`)
+* `b/BLOOD_TYPE`: Matches the blood type field; The only valid keywords for this field are `A/B/AB/O`
+  (Valid for: `Patients`)
+* `m/MEDICAL_CONDITION`: Limits the keyword search to the list of medical conditions of a patient (Valid
```

Comment:
Should we change this parameter flag to `o/` to be consistent with the `add patient` command?

```suggestion
* `o/MEDICAL_CONDITION`: Limits the keyword search to the list of medical conditions of a patient (Valid
```

Reply:
this sounds good